torch/lib/c10d/reducer.cpp

```diff
@@ -22,6 +22,15 @@ inline int64_t current_time_in_nanos() {
   return torch::autograd::profiler::getTime();
 }
 
+std::string parse_env(const char* env_var_name) {
```

Comment:
parseEnvVarFlag is not a generic parsing function, which specifically parse values as True/1 or False/0. Also, it may throw exceptions if the values are not 1 or 0.